frontend/src/features/accounts/sign-up-form.ts

```diff
@@ -254,7 +268,11 @@ export class SignUpForm extends LiteElement {
         try {
           await this.logIn({ email, password });
         } catch {
-          this.dispatchEvent(new CustomEvent("unauthenticated"));
+          this.serverError = msg(
+            "User is already registered, but with a different password.",
+          );
```

Comment:
```suggestion
          this.serverError = msg("User is already registered.");
```

We don't need to tell the user about whether their password is right at this stage I think, can just direct them to login and use the functionality that already exists there. Otherwise this could become an easy means by which someone could phish for user passwords.

Reply:
The current behavior is if the password is correct, it will auto log them in - maybe that's confusing, but not sure if we want to refactor that now? But I guess this still keeps it fairly vague, saying that they need to log in because user is already registered.

Reply:
Yeah tbh it's probably fine for now.